test/grouping.jl

```diff
@@ -25,7 +25,7 @@ module TestGrouping
     byf = by(dt, :a, dt -> DataTable(bsum = sum(dt[:b])))
 
     @test all(T -> T <: AbstractVector, map(typeof, colwise([sum], dt)))
-    @test all(T -> T <: AbstractVector, map(typeof, colwise(sum, dt)))
+    @test all(T -> T <: Nullable, map(typeof, colwise(sum, dt)))
```

Comment:
Don't we have any tests which also check the value of the result? Would make sense to add one of two.